example/app.android.js

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable no-console */
+/**
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import React from 'react';
+import { StyleSheet, View, Image, Text } from 'react-native';
+import { AppleButton, appleAuthAndroid } from '@invertase/react-native-apple-authentication';
+import appleLogoWhite from './images/apple_logo_white.png';
+import appleLogoBlack from './images/apple_logo_black.png';
+
+
+export default RootComponent = () => {
+  const doAppleLogin = async () => {
+    try {
+      // Initialize the module
+      appleAuthAndroid.configure({
+        clientId: "Your client ID",
```

Comment:
If we could be prescriptive about where to get the id like below it would decrease future support:

```suggestion
        clientId: "Your client ID",  // <-- get this client id from ... ?, should look like 'asdfsfdasdfasdf'
```